app/config/sql/board.sql

```diff
@@ -1,27 +1,62 @@
---
--- Create database
---
+--Create Database
 CREATE DATABASE IF NOT EXISTS board;
-GRANT SELECT, INSERT, UPDATE, DELETE ON board.* TO board_root@localhost IDENTIFIED BY ‘board_root’;
+GRANT SELECT, INSERT, UPDATE, DELETE ON forum.* TO board_root@localhost IDENTIFIED BY 'board_root';
 FLUSH PRIVILEGES;
---
--- Create tables
---
+
+--Create Tables
 USE board;
-CREATE TABLE IF NOT EXISTS thread (
-id INT UNSIGNED NOT NULL AUTO_INCREMENT,
-title VARCHAR(255) NOT NULL,
-created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-PRIMARY KEY (id)
+
+--User Table
+CREATE TABLE IF NOT EXISTS user (
+     id INT(11) NOT NULL AUTO_INCREMENT,
+     username VARCHAR(50) NOT NULL,
+     first_name VARCHAR(50) NOT NULL,
+     last_name VARCHAR(50) NOT NULL,
+     email VARCHAR(255) NOT NULL, 
```

Comment:
This column name seems to be misleading po :sweat: 
it could either thought to be the email (message in email) or email address ..
please be specific po :bow: